schema/schema.go

```diff
@@ -107,8 +107,8 @@ func GetCollectionType(reqSchema jsoniter.RawMessage) (CollectionType, error) {
 		switch string(val) {
 		case "documents":
 			return DocumentsType, nil
-		case "messages":
```

Comment:
Do you want to keep this check for backwards compatibility?

Reply:
Okay, let me keep it, and will remove it in the second round.